### PR TITLE
Update loc xaml

### PR DESCRIPTION
**Pull request by nguerrera**
**Customer scenario**

See #1472, the changes did not apply to languages other then English so all of those issues would remain for non-English VS users

**Bugs this fixes:**

Same as #1472, but for all non-English languages.

**Workarounds, if any**

None

**Risk**

Low

**Performance impact**

None

**Is this a regression from a previous update?**

No

**Root cause analysis:**

The loc process for these files currently requires a manual step due to lack of resources and late onboarding of new loc process with no pre-baked tooling we could use. We are planning to automate everything for next cycle in partnership with central engineering team. Sri and I had been doing a good job pointing out when the tool had to be re-run but missed it on this PR.

**How was the bug found?**

I saw a CHS only bug report incoming this monrning ([VSO 379990](https://devdiv.visualstudio.com/DevDiv/_workitems?id=379990&_a=edit)) and thought it might be due to something wrong in these files (it may still be - investigating separately) but that's when I spotted these were missed with the most recent xaml change. 

@333fred @srivatsn

### CODE REVIEW

**Comment by nguerrera** [comment]
@MattGertz for approval

**Comment by natidea** [comment]
Approved to merge via [381803](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/381803)

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ru/ResolvedSdkReference.xaml [hunk 1]
```diff
@@ -13,8 +13,8 @@
   <StringProperty Name="Name" DisplayName="Name" ReadOnly="True" />
   <StringProperty Name="Version" DisplayName="Version" ReadOnly="True" />
   <StringProperty Name="SDKPackageItemSpec" Visible="False" ReadOnly="True" />
-  <BoolProperty Name="CopyPayload" DisplayName="Копировать полезные данные" />
-  <StringProperty Name="CopyPayloadToSubDirectory" DisplayName="Копировать полезные данные в подкаталог" ReadOnly="True" />
+  <BoolProperty Name="CopyPayload" DisplayName="Копировать полезные данные" ReadOnly="True" />
+  <StringProperty Name="CopyPayloadToSubDirectory" DisplayName="Copy Payload to Subdirectory" ReadOnly="True" />
```
Comment (nguerrera) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ru/ResolvedSdkReference.xaml]: Hold on merging. Investigating why this changed.

Comment (nguerrera) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ru/ResolvedSdkReference.xaml]: It accurately reflects the state of the source code. It is untranslated in xlf and untranslated in other languages as it was added to this file after last loc handback. Seems like it was manually translated in c462e48.

**Pull request merged**